<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>航班购票系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/imagSoure/sysu.ico</normaloff>:/imagSoure/sysu.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	
	background-image: url(:/imagSoure/flightLogin.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>421</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="loginWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>371</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#loginWidget{
	
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>310</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton{
	background-color: rgb(0, 170, 255);
	border-radius: 10px;
	font-size: 14px; 
}
#pushButton:hover{
	
	background-color: rgb(38, 143, 255);
	font-size: 14px; 
	border-radius: 10px;
}
#pushButton:pressed {
    background-color: rgb(38, 143, 255);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px;  /* 鼠标按下时，字体缩小 */
}</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>241</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="username">
          <property name="styleSheet">
           <string notr="true">#username{
  height: 30px;
}</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password">
          <property name="styleSheet">
           <string notr="true">#password{
  height: 30px;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>请输入密码</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>-20</y>
         <width>281</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label{
	image: url(:/imagSoure/title.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>330</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="forget">
          <property name="styleSheet">
           <string notr="true">#forget{
	
	color: rgb(0, 85, 255);
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
}
#forget:hover{
	
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	text-decoration: underline;
}</string>
          </property>
          <property name="text">
           <string>忘记密码</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="register_2">
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#register_2{
	background-color: rgb(255, 255, 255);
	
	color: rgb(0, 85, 255);
	border-radius: 10px;
}
#register_2:hover{
	
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	text-decoration: underline;
}
</string>
          </property>
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="registerWidget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>371</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#registerWidget{
	
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>-20</y>
         <width>281</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_2{
	image: url(:/imagSoure/title.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>241</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="registerUsername">
          <property name="styleSheet">
           <string notr="true">#registerUsername{
	height: 30px;
}</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="registerPassword">
          <property name="styleSheet">
           <string notr="true">#registerPassword{
	height: 30px;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>请输入密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="registerComfirPass">
          <property name="styleSheet">
           <string notr="true">#registerComfirPass{
	height: 30px;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>再次确认密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="comfirProblem">
          <property name="styleSheet">
           <string notr="true">#comfirProblem{
	height: 30px;
}</string>
          </property>
          <property name="placeholderText">
           <string>你最喜欢的大学是？</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="zhuce">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>350</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#zhuce{
	
	background-color: rgb(0, 170, 255);
	font-size: 14px; 
	border-radius: 10px;
}
#zhuce:hover{
	
	background-color: rgb(38, 143, 255);
	font-size: 14px; 
	border-radius: 10px;
}
#zhuce:pressed{
	background-color: rgb(38, 143, 255);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px; 
}</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="back1">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>390</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#back1{
	background-color: rgb(0, 170, 255);
	font-size: 14px;
	border-radius: 10px;
}
#back1:hover{
	
	background-color: rgb(38, 143, 255);
	font-size: 14px;
	border-radius: 10px;
}

#back1:pressed{
	background-color: rgb(38, 143, 255);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px; 
}</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="forgetWidget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>371</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#forgetWidget{
	
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>-20</y>
         <width>281</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_3{
	image: url(:/imagSoure/title.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>241</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="forgetUsername">
          <property name="styleSheet">
           <string notr="true">#forgetUsername{
  height: 30px;
}</string>
          </property>
          <property name="placeholderText">
           <string>请输入用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="forgetProblem">
          <property name="styleSheet">
           <string notr="true">#forgetProblem{
  height: 30px;
}</string>
          </property>
          <property name="placeholderText">
           <string>你最喜欢的大学是？</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="forgetPassword">
          <property name="styleSheet">
           <string notr="true">#forgetPassword{
  height: 30px;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>请输入新密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="forgetComfirPassword">
          <property name="styleSheet">
           <string notr="true">#forgetComfirPassword{
  height: 30px;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>再次确认新密码</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="chongzhi">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>350</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#chongzhi{
	background-color: rgb(0, 170, 255);
	font-size:14px;
	border-radius: 10px;
}
#chongzhi:hover{
	
	background-color: rgb(38, 143, 255);
	font-size:14px;
	border-radius: 10px;
}
#chongzhi:pressed{
	background-color: rgb(38, 143, 255);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px; 
}</string>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
      <widget class="QPushButton" name="back2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>390</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#back2{
	background-color: rgb(0, 170, 255);
	font-size:14px;
	border-radius: 10px;
}
#back2:hover{
	
	background-color: rgb(38, 143, 255);
	font-size: 14px;
	border-radius: 10px;
}
#back2:pressed{
	background-color: rgb(38, 143, 255);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px; 
}</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
